### ISSUE TITLE
java.io.FileNotFoundException: Could not locate clj_kondo/main__init.class

### ISSUE THREAD

**Issue by anderseknert** [clj-kondo/lein-clj-kondo#1]
if I use `:plugins [[clj-kondo/lein-clj-kondo "0.1.1"]]` as instructed in the README, it says it can't find the plugin.. adding com.github in front seems to locate the plugin (so I guess the README has it wrong?), but then I'm instead presented with the stacktrace below:

```
lein clj-kondo
clojure.lang.Compiler$CompilerException: Syntax error compiling at (leiningen/clj_kondo.clj:1:1).
#:clojure.error{:phase :compile-syntax-check, :line 1, :column 1, :source "leiningen/clj_kondo.clj"}
 at clojure.lang.Compiler.load (Compiler.java:7652)
    clojure.lang.RT.loadResourceScript (RT.java:381)
    clojure.lang.RT.loadResourceScript (RT.java:372)
    clojure.lang.RT.load (RT.java:459)
    clojure.lang.RT.load (RT.java:424)
    clojure.core$load$fn__6856.invoke (core.clj:6115)
    clojure.core$load.invokeStatic (core.clj:6114)
    clojure.core$load.doInvoke (core.clj:6098)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.core$load_one.invokeStatic (core.clj:5897)
    clojure.core$load_one.invoke (core.clj:5892)
    clojure.core$load_lib$fn__6796.invoke (core.clj:5937)
    clojure.core$load_lib.invokeStatic (core.clj:5936)
    clojure.core$load_lib.doInvoke (core.clj:5917)
    clojure.lang.RestFn.applyTo (RestFn.java:142)
    clojure.core$apply.invokeStatic (core.clj:669)
    clojure.core$load_libs.invokeStatic (core.clj:5974)
    clojure.core$load_libs.doInvoke (core.clj:5958)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invokeStatic (core.clj:669)
    clojure.core$require.invokeStatic (core.clj:5996)
    clojure.core$require.doInvoke (core.clj:5996)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    leiningen.core.utils$require_resolve.invokeStatic (utils.clj:102)
    leiningen.core.utils$require_resolve.invoke (utils.clj:95)
    leiningen.core.utils$require_resolve.invokeStatic (utils.clj:105)
    leiningen.core.utils$require_resolve.invoke (utils.clj:95)
    leiningen.core.main$lookup_task_var.invokeStatic (main.clj:69)
    leiningen.core.main$lookup_task_var.invoke (main.clj:65)
    leiningen.core.main$pass_through_help_QMARK_.invokeStatic (main.clj:79)
    leiningen.core.main$pass_through_help_QMARK_.invoke (main.clj:73)
    leiningen.core.main$task_args.invokeStatic (main.clj:82)
    leiningen.core.main$task_args.invoke (main.clj:81)
    leiningen.core.main$resolve_and_apply.invokeStatic (main.clj:339)
    leiningen.core.main$resolve_and_apply.invoke (main.clj:336)
    leiningen.core.main$_main$fn__7429.invoke (main.clj:453)
    leiningen.core.main$_main.invokeStatic (main.clj:442)
    leiningen.core.main$_main.doInvoke (main.clj:439)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.lang.Var.applyTo (Var.java:705)
    clojure.core$apply.invokeStatic (core.clj:667)
    clojure.main$main_opt.invokeStatic (main.clj:514)
    clojure.main$main_opt.invoke (main.clj:510)
    clojure.main$main.invokeStatic (main.clj:664)
    clojure.main$main.doInvoke (main.clj:616)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.lang.Var.applyTo (Var.java:705)
    clojure.main.main (main.java:40)
Caused by: java.io.FileNotFoundException: Could not locate clj_kondo/main__init.class, clj_kondo/main.clj or clj_kondo/main.cljc on classpath. Please check that namespaces with dashes use underscores in the Clojure file name.
 at clojure.lang.RT.load (RT.java:462)
    clojure.lang.RT.load (RT.java:424)
    clojure.core$load$fn__6856.invoke (core.clj:6115)
    clojure.core$load.invokeStatic (core.clj:6114)
    clojure.core$load.doInvoke (core.clj:6098)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.core$load_one.invokeStatic (core.clj:5897)
    clojure.core$load_one.invoke (core.clj:5892)
    clojure.core$load_lib$fn__6796.invoke (core.clj:5937)
    clojure.core$load_lib.invokeStatic (core.clj:5936)
    clojure.core$load_lib.doInvoke (core.clj:5917)
    clojure.lang.RestFn.applyTo (RestFn.java:142)
    clojure.core$apply.invokeStatic (core.clj:669)
    clojure.core$load_libs.invokeStatic (core.clj:5974)
    clojure.core$load_libs.doInvoke (core.clj:5958)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invokeStatic (core.clj:669)
    clojure.core$require.invokeStatic (core.clj:5996)
    clojure.core$require.doInvoke (core.clj:5996)
    clojure.lang.RestFn.invoke (RestFn.java:436)
    leiningen.clj_kondo$eval663$loading__6737__auto____664.invoke (clj_kondo.clj:1)
    leiningen.clj_kondo$eval663.invokeStatic (clj_kondo.clj:1)
    leiningen.clj_kondo$eval663.invoke (clj_kondo.clj:1)
    clojure.lang.Compiler.eval (Compiler.java:7181)
    clojure.lang.Compiler.eval (Compiler.java:7170)
    clojure.lang.Compiler.load (Compiler.java:7640)
    clojure.lang.RT.loadResourceScript (RT.java:381)
    clojure.lang.RT.loadResourceScript (RT.java:372)
    clojure.lang.RT.load (RT.java:459)
    clojure.lang.RT.load (RT.java:424)
    clojure.core$load$fn__6856.invoke (core.clj:6115)
    clojure.core$load.invokeStatic (core.clj:6114)
    clojure.core$load.doInvoke (core.clj:6098)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.core$load_one.invokeStatic (core.clj:5897)
    clojure.core$load_one.invoke (core.clj:5892)
    clojure.core$load_lib$fn__6796.invoke (core.clj:5937)
    clojure.core$load_lib.invokeStatic (core.clj:5936)
    clojure.core$load_lib.doInvoke (core.clj:5917)
    clojure.lang.RestFn.applyTo (RestFn.java:142)
    clojure.core$apply.invokeStatic (core.clj:669)
    clojure.core$load_libs.invokeStatic (core.clj:5974)
    clojure.core$load_libs.doInvoke (core.clj:5958)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invokeStatic (core.clj:669)
    clojure.core$require.invokeStatic (core.clj:5996)
    clojure.core$require.doInvoke (core.clj:5996)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    leiningen.core.utils$require_resolve.invokeStatic (utils.clj:102)
    leiningen.core.utils$require_resolve.invoke (utils.clj:95)
    leiningen.core.utils$require_resolve.invokeStatic (utils.clj:105)
    leiningen.core.utils$require_resolve.invoke (utils.clj:95)
    leiningen.core.main$lookup_task_var.invokeStatic (main.clj:69)
    leiningen.core.main$lookup_task_var.invoke (main.clj:65)
    leiningen.core.main$pass_through_help_QMARK_.invokeStatic (main.clj:79)
    leiningen.core.main$pass_through_help_QMARK_.invoke (main.clj:73)
    leiningen.core.main$task_args.invokeStatic (main.clj:82)
    leiningen.core.main$task_args.invoke (main.clj:81)
    leiningen.core.main$resolve_and_apply.invokeStatic (main.clj:339)
    leiningen.core.main$resolve_and_apply.invoke (main.clj:336)
    leiningen.core.main$_main$fn__7429.invoke (main.clj:453)
    leiningen.core.main$_main.invokeStatic (main.clj:442)
    leiningen.core.main$_main.doInvoke (main.clj:439)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.lang.Var.applyTo (Var.java:705)
    clojure.core$apply.invokeStatic (core.clj:667)
    clojure.main$main_opt.invokeStatic (main.clj:514)
    clojure.main$main_opt.invoke (main.clj:510)
    clojure.main$main.invokeStatic (main.clj:664)
    clojure.main$main.doInvoke (main.clj:616)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.lang.Var.applyTo (Var.java:705)
    clojure.main.main (main.java:40)
```

Running clj-kondo with the plugin configured in ~/.lein/profiles.clj works as expected.

I don't do a whole lot of Clojure development, so excuse me if I'm missing something obvious 😅

**Comment by ericdallo** [comment]
@anderseknert thanks, I fixed the readme.
Also, I released a new version bumping clj-kondo version, could you test with 0.1.2?
If that doesn't fix, could you past your project.clj ?

**Comment by anderseknert** [comment]
Thanks for taking a look @ericdallo 👍 I think you acidentally added a `v` prepended to the last version :) But once I figured that out, the problem unfortunately persists. 

My project.clj can be found [here](https://github.com/anderseknert/clj-opa).

**Comment by ericdallo** [comment]
> Thanks for taking a look @ericdallo +1 I think you acidentally added a v prepended to the last version :) But once I figured that out, the problem unfortunately persists.

Oh, good finding 🤦🏻‍♂️

> My project.clj can be found here.

will take a look

**Comment by ericdallo** [comment]
Fixed on 0.1.3 @anderseknert , it was a issue with CI deploying to clojars a invalid jar

**Comment by anderseknert** [comment]
Thanks @ericdallo! Works like a charm now 😃
